transform/schemas/properties/relations.py

```diff
@@ -0,0 +1,63 @@
+import os
+from schemas.properties.data import ID, LEGALNAME
+from schemas.properties.env import *
+from utils.loader import load_env_vars
+
+env_vars = load_env_vars()
+
+# HELPER COLUM NAMES
+DIRECTORY = "directory"
+FILE = "file"
+SOURCE = "source"
+SOURCE_FILE_PATH = "source_file_path"
+SOURCE_TYPE = "source_type"
+TARGET = "target"
+TARGET_FILE_PATH = "target_file_path"
+TARGET_TYPE = "target_type"
+
+# DICT KEYS
+DATASET_KEY = "dataset"
+ORGANIZATION_KEY = "organization"
+OTHERRESEARCHPRODUCT_KEY = "otherresearchproduct"
+PUBLICATION_KEY = "publication"
+PROJECT_KEY = "project"
+SOFTWARE_KEY = "software"
+
+
+# AGGREGATED COLUMNS -> these columns are added to each other, so can not be tuples
+SELECTED_COLUMNS = [ID]
+ADDITIONAL_COLUMNS = [LEGALNAME]
+
+# PARQUET FILES
+COMBINED_PQ_FILE = os.path.join(env_vars[OUTPUT_PATH], "combined_data.parquet")
```

Comment:
The possibility of creating a local dump is deprecated. Therefore, OUTPUT_PATH shouldnt be used anymore. I believe we can make those paths static and place them into e.g. tmp/ - we can hardcode those paths

Reply:
BTW. 
- Do we delete those files after the update? 
- Shouldnt we do so?

Reply:
Added `clean_tmp` into utils.utils